src/components/Topbar/Topbar.jsx

```diff
@@ -37,17 +40,38 @@ export const Topbar = () => {
           aria-expanded={open ? 'true' : undefined}
           onClick={handleClick}
         >
-          Lesson Plan
+          Facilitation Resources
         </NavbarText>
         <Menu
           id='basic-menu'
           anchorEl={anchorEl}
           open={open}
           onClose={handleClose}
         >
-          <MenuItem onClick={handleClose}>Classroom</MenuItem>
-          <MenuItem onClick={handleClose}>Corporate</MenuItem>
-          <MenuItem onClick={handleClose}>Youth Engagement</MenuItem>
+          <MenuItem
+            onClick={handleClose}
+            href={
+              'https://docs.google.com/presentation/d/1XsiCXh4mgDa4O470tRjYaThRSJ4lAQ7HjrPzrMhvRZc/edit?usp=sharing'
+            }
+          >
+            Classroom Lesson Plan
+          </MenuItem>
+          <MenuItem
+            onClick={handleClose}
+            href={
+              'https://docs.google.com/presentation/d/1XsiCXh4mgDa4O470tRjYaThRSJ4lAQ7HjrPzrMhvRZc/edit?usp=sharing'
+            }
+          >
+            Youth Engagement (Pending)
+          </MenuItem>
+          <MenuItem
+            onClick={handleClose}
+            href={
+              'https://docs.google.com/presentation/d/1XsiCXh4mgDa4O470tRjYaThRSJ4lAQ7HjrPzrMhvRZc/edit?usp=sharing'
+            }
+          >
+            Corporate Training (Pending)
+          </MenuItem>
```

Comment:
The links don't appear to work yet when we click on them! we could add an extra `component={Link}` (importing `Link` from `@mui/material`) to get these to work:

```suggestion
          <MenuItem
            onClick={handleClose}
            href={
              'https://docs.google.com/presentation/d/1XsiCXh4mgDa4O470tRjYaThRSJ4lAQ7HjrPzrMhvRZc/edit?usp=sharing'
            }
            component={Link}
          >
            Classroom Lesson Plan
          </MenuItem>
          <MenuItem
            onClick={handleClose}
            href={
              'https://docs.google.com/presentation/d/1XsiCXh4mgDa4O470tRjYaThRSJ4lAQ7HjrPzrMhvRZc/edit?usp=sharing'
            }
            component={Link}
          >
            Youth Engagement (Pending)
          </MenuItem>
          <MenuItem
            onClick={handleClose}
            href={
              'https://docs.google.com/presentation/d/1XsiCXh4mgDa4O470tRjYaThRSJ4lAQ7HjrPzrMhvRZc/edit?usp=sharing'
            }
            component={Link}
          >
            Corporate Training (Pending)
          </MenuItem>
```

Reply:
Thanks - let's go with that1